<launch>
	<node name="joint_action_server" pkg="cs4752_proj2" type="joint_action_server.py" output="screen"></node>
	<node name="test_line" pkg="cs4752_proj2" type="test-line.py" output="screen"></node>
	<node name="robot_interface" pkg="cs4752_proj2" type="robot_interface.py" output="screen"></node>
</launch>
